app.js

```diff
@@ -0,0 +1,43 @@
+var express = require('express');
+var path = require('path');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
```

Comment:
There will be no need to parse POST requests because no POST requests will be made